manual/statements/batch/README.md

```diff
@@ -1,5 +1,32 @@
 ## Batch statements
 
-*Coming soon... In the meantime, see the javadoc for [BatchStatement].*
+Use [BatchStatement] to execute a set of queries as an atomic operation (refer to
```

Comment:
1. Do we want to explain batch types (unlogged vs logged vs counter)?
2. I think that "atomic" is only correct for a batch if all statements share the same partition key.

Reply:
I replaced 'atomic' with 'single'.  

I also added a sentence about using `UNLOGGED`, I didn't mention `COUNTER` here its advanced and covered in CQL documentation, so should be apparent how to use it if needed.

Reply:
If you are ok with these changes I will follow on and also make them on 4.x.